### PR TITLE
Add Sphinx template

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

File: template/{% if documentation == 'sphinx' %}docs{% endif %}/index.rst.jinja
Comment (sneakers-the-rat) [inline template/{% if documentation == 'sphinx' %}docs{% endif %}/index.rst.jinja:1]: thoughts on using myst ubiquitously in the docs?

Comment (blink1073) [reply template/{% if documentation == 'sphinx' %}docs{% endif %}/index.rst.jinja:1]: done!

**Review by lwasser — APPROVED** [review]
If everyone is good with this pr I certainly am!! please merge! THANK You for this. i can't believe how quickly this template came together and am so impressed by the team work involved in making it happen.

File: tests/test_template_init.py [hunk 1]
```diff
@@ -49,6 +49,15 @@ def license(request: pytest.FixtureRequest) -> str:
     return request.param
 
 
+@pytest.fixture(
```
[the remark this answers is not in the document]
Comment (blink1073) [reply tests/test_template_init.py:52]: It creates versions of the test for each combination of inputs.

File: tests/test_template_init.py [hunk 2]
```diff
@@ -49,6 +49,15 @@ def license(request: pytest.FixtureRequest) -> str:
...
+    scope="module",
+    params=["mkdocs", "sphinx", ""],
+)
+def documentation(request: pytest.FixtureRequest) -> str:
+    """Provide a recognized license classification."""
```
Comment (Midnighter) [inline tests/test_template_init.py:57]:
```suggestion
    """Provide a documentation option."""
```

**Review by Midnighter — APPROVED** [review]
Awesome, thanks for the quick work @blink1073.

I didn't check the sphinx config, since I have little experience with that.

What was your reason for dropping Python 3.9? It's still supported for another year. Anyway, I'm fine with doing so 🙂, it shouldn't obstruct a merge.

**Comment by blink1073** [comment]
The recent major versions of Sphinx and myst_parser dropped support for Python 3.9.

**Pull request merged**